### ISSUE TITLE
New project create, got esliont failure

### ISSUE THREAD

**Issue by ktwbc** [swarmion/swarmion#863]
### Describe the bug

output from brand new project create:
```
Linting your project...


Aborting installation.
Unexpected error. Please report it as a bug:
Error: Command failed: pnpm lint-fix-all
    at checkExecSyncError (node:child_process:890:11)
    at execSync (node:child_process:962:15)
    at /Volumes/dev/.pnpm-store/v3/tmp/dlx-11602/node_modules/.pnpm/create-swarmion-app@0.31.0/node_modules/create-swarmion-app/dist/index.js:30:340436
    at Generator.next (<anonymous>)
    at fulfilled (/Volumes/dev/.pnpm-store/v3/tmp/dlx-11602/node_modules/.pnpm/create-swarmion-app@0.31.0/node_modules/create-swarmion-app/dist/index.js:30:338718)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 12226,
  stdout: null,
  stderr: null
}
```

### Reproduction

simply ran "pnpm create swarmion-app"

### System Info

```shell
System:
    OS: macOS 14.2.1
    CPU: (10) arm64 Apple M1 Max
    Memory: 2.47 GB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.19.0 - ~/.nvm/versions/node/v18.19.0/bin/node
    Yarn: 1.22.19 - ~/.yarn/bin/yarn
    npm: 9.6.6 - /opt/homebrew/bin/npm
    pnpm: 8.14.0 - /opt/homebrew/bin/pnpm
  Browsers:
    Chrome: 120.0.6099.216
    Safari: 17.2.1
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/swarmion/swarmion/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://www.swarmion.dev/docs/).
- [X] Check that there isn't [already an issue](https://github.com/swarmion/swarmion/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by fargito** [comment]
Hello @ktwbc, I'm trying to investigate, what starting template did you choose?

**Comment by ktwbc** [comment]
I actually I got this on both  Swarmion Starter and Serverless framework full stack

**Comment by fargito** [comment]
Hello @ktwbc I think this is due to a bad build, It should be fixed in the next release

**Comment by fargito** [comment]
I have published v0.31.2, the issue should be fixed, can you confirm?
